parlai/tasks/anli/anli_constants.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+ANLI = 'ANLI'
+ANLI_VERSION = 'v0.1'
+ANLI_PREFIX = 'anli_'
```

Comment:
nice!

Reply:
> one nit, but otherwsie _awesome_ job 😄
> 
> quick question -- is this the same output format that is returned by the `snli` and `mnli` tasks in ParlAI? if not we should standardize them for easy multi-tasking.

Great question!! Here is some snippet of output from snli and mnli:

```
- - - - - - - - - - - - - - - - - - - - -
~~
[SNLI]: Premise: A boy is jumping on skateboard in the middle of a red bridge.
Hypothesis: The boy is wearing safety equipment.
[label_candidates: contradiction|entailment|neutral]
[labels: neutral]
- - - - - - - - - - - - - - - - - - - - -
~~
[SNLI]: Premise: An older man sits with his orange juice at a small table in a coffee shop while employees in bright colored shirts smile in the background.
Hypothesis: An older man drinks his juice as he waits for his daughter to get off work.
[label_candidates: contradiction|entailment|neutral]
[labels: neutral]
- - - - - - - - - - - - - - - - - - - - -
~~
[ loaded 549367 episodes with a total of 549367 examples ]
```
```
- - - - - - - - - - - - - - - - - - - - -
~~
[MultiNLI]: Premise: Gays and lesbians.
Hypothesis: Heterosexuals.
[label_candidates: entailment|contradiction|neutral]
[labels: contradiction]
- - - - - - - - - - - - - - - - - - - - -
~~
[MultiNLI]: Premise: At the end of Rue des Francs-Bourgeois is what many consider to be the city's most handsome residential square, the Place des Vosges, with its stone and red brick facades.
Hypothesis: Place des Vosges is constructed entirely of gray marble.
[label_candidates: entailment|contradiction|neutral]
[labels: contradiction]
- - - - - - - - - - - - - - - - - - - - -
~~
[ loaded 392702 episodes with a total of 392702 examples ]
```

running multi-task: ```  python examples/display_data.py -t multinli,snli,anli ```
```
- - - - - - - - - - - - - - - - - - - - -
~~
loading: /Users/jingxu23/workplace/fbgithub/ParlAI/data/ANLI/anli_v0.1/R2/train.jsonl
[world 2:anli:r1, anli:r2, anli:r3]
[ANLI:R2]: Premise: Topi Raja Sweety Roja is a 1996 Telugu comedy film, produced by Dr. A. Vijayalakshmi on Sri Sai Madhavi Productions banner and directed by Dr. N. Siva Prasad. Starring Rajendra Prasad, Roja in the lead roles and music also composed by "Hero" Rajendra Prasad.
Hypothesis: The lead role was played by Rajendra Prasad
[label_candidates: neutral|entailment|contradiction]
[labels: entailment]
- - - - - - - - - - - - - - - - - - - - -
~~
[world 0:MultiNLI]
[MultiNLI]: Premise: One of our number will carry out your instructions minutely.
Hypothesis: A member of my team will execute your orders with immense precision.
[label_candidates: entailment|contradiction|neutral]
[labels: entailment]
- - - - - - - - - - - - - - - - - - - - -
~~
[world 1:SNLI]
[SNLI]: Premise: A person on a horse jumps over a broken down airplane.
Hypothesis: A person is outdoors, on a horse.
[label_candidates: neutral|entailment|contradiction]
[labels: entailment]
- - - - - - - - - - - - - - - - - - - - -
~~
loading: /Users/jingxu23/workplace/fbgithub/ParlAI/data/ANLI/anli_v0.1/R3/train.jsonl
[world 2:anli:r1, anli:r2, anli:r3]
[ANLI:R3]: Premise: TOKYO, Dec 18 (Reuters) - Japan’s Shionogi & Co said on Tuesday that it has applied to health regulators in the United States, Canada and Europe for approval of its HIV drug Dolutegravir. Shionogi developed Dolutegravir with a Viiv Healthcare, an AIDS drug joint venture between GlaxoSmithKline and Pfizer, in exchange for its rights to the drug.
Hypothesis: The article was written on December 18th.
[label_candidates: neutral|entailment|contradiction]
[labels: entailment]
- - - - - - - - - - - - - - - - - - - - -
~~
[world 0:MultiNLI]
[MultiNLI]: Premise: How do you know? All this is their information again.
Hypothesis: This information belongs to them.
[label_candidates: entailment|contradiction|neutral]
[labels: entailment]
- - - - - - - - - - - - - - - - - - - - -
~~
[ loaded 1104934 episodes with a total of 1104934 examples ]
```